### PR TITLE
fix: #548 get rid of errors in thr console

### PR DESCRIPTION
**Pull request by AKiroi**
get rid of errors in thr console

### CODE REVIEW

**Review by Ekaterina1994 — COMMENTED** [review]
Please check all pages in console. I remember that there were also other errors)

File: mw-webapp/src/hooks/usePersistanceState.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const usePersistanceState = <
 
   useEffect(() => {
     setValue(validatedStoredValue);
-  }, [params.storedDataValidator]);
```
Comment (tritonJS826) [inline mw-webapp/src/hooks/usePersistanceState.tsx:66]: Are you sure nothing is broken?

**Review by tritonJS826 — CHANGES_REQUESTED** [review]

**Comment by tritonJS826** [comment]
Looks like these changes broke business logic. Be careful. I will send you a video

**Comment by AKiroi** [comment]
fixex this mistakes 
image

**Comment by AKiroi** [comment]
need to add typization to dependencies in usePersistanceState

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**